### ISSUE TITLE
Some builds not shown

### ISSUE THREAD

**Issue by DuvG62** [amoerie/teamcity-theatre#78]
Hi and thanks for this project - the inclusion of branches is a total winner for me  :)

But... I'm running version 3.0.2 and for some reason only the builds which show a branch name on the TeamCity Overview page are being picked up.  Any build which shows either a build #number or `<default>`  where TeamCity often shows a branch name are not shown at all on the Dashboard.

Investigations show that the TC API is not returning those builds.  I looked at the request made (thanks for the verbose logging!)  and it seems that if I remove 'policy:active_history_and_active_vcs_branches' from the request that goes to the TC API then my missing builds are now returned.

So I wanted to try removing that part from the requests that go to the TC API but I can't find any reference to this policy filter anywhere in the source???

Any ideas?

**Comment by amoerie** [comment]
Hi DuvG62 and thanks for the kind words!

I’m planning to make the query configurable in the appsettings.json file,
but for now you’ll find it hardcoded in this file:
https://github.com/amoerie/teamcity-theatre/blob/master/src/TeamCityTheatre.Core/DataServices/BuildDataService.cs

Let me know if that answers your question :-)

**Comment by DuvG62** [comment]
Thanks Alexander - that steered me in the right direction (although I was never able to find where `policy:active_history_and_active_vcs_branches` was coming from).

I tried building your latest source code but wwwroot and appsettings.json are always missing from my publish-output folder even though I have all dependencies.  In the end I've made a bit of a hack by copying in the above two and web.config from the 3.0.2 release version, then it all runs and I have my missing builds!  :)

I'll grab your new version when it's released so I can replace that hack.

Thanks again,

Duv
